public/utils.js

```diff
@@ -120,6 +120,11 @@ define(function (require) {
       }
       return aggConfig;
     },
+    isXYZurl: function (url) {
```

Comment:
can we rename to proper camel case 
```suggestion
    isXyzUrl: function (url) {
```